compiler/backend/src/org/jetbrains/kotlin/codegen/inline/LambdaInfo.kt

```diff
@@ -256,7 +254,7 @@ class PsiExpressionLambda(
 
     val functionWithBodyOrCallableReference: KtExpression = (expression as? KtLambdaExpression)?.functionLiteral ?: expression
 
-    private val labels: Set<String>
+    override val returnLabels: Map<String, Label?>
```

Comment:
Maybe best to keep it private and just add new method here  to fill labels map required for IrSourceCompilerForInline?

Reply:
TBH that seems like an unnecessarily verbose way to achieve the same result, since both the field and the map are immutable anyway